js/addToTask.js

```diff
@@ -66,3 +67,12 @@ TRASH_INPUT.addEventListener(CLICK_EVENT, function (event) {
   INPUT_TASK.classList.add(HIDE);
   INPUT.value = null;
 });
+
+UL_LIST.addEventListener(CLICK_EVENT, function (event) {
```

Comment:
see https://github.com/rsa-wtag/Todo-App-with-JS/pull/2#discussion_r1161261757

Reply:
the anonymous arrow function should be extracted to a named function and use it for both event registration and unregistration.

```
function onRemoveTodo() {
    ...
}

UL_LIST.addEventListener(CLICK_EVENT, onRemoveTodo)
UL_LIST. removeEventListener(CLICK_EVENT, onRemoveTodo)
```